\doxysection{Command\+Error Class Reference}
\hypertarget{classpip_1_1__internal_1_1exceptions_1_1CommandError}{}\label{classpip_1_1__internal_1_1exceptions_1_1CommandError}\index{CommandError@{CommandError}}


Inheritance diagram for Command\+Error\+:\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=170pt]{d9/d7d/classpip_1_1__internal_1_1exceptions_1_1CommandError__inherit__graph}
\end{center}
\end{figure}


Collaboration diagram for Command\+Error\+:\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=170pt]{dc/d31/classpip_1_1__internal_1_1exceptions_1_1CommandError__coll__graph}
\end{center}
\end{figure}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Raised when there is an error in command-line arguments\end{DoxyVerb}
 

Definition at line \mbox{\hyperlink{__internal_2exceptions_8py_source_l00286}{286}} of file \mbox{\hyperlink{__internal_2exceptions_8py_source}{exceptions.\+py}}.



The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
/home/liwuen/projects\+\_\+dev/pqc-\/engineering-\/ssec-\/23/venv/lib/python3.\+12/site-\/packages/pip/\+\_\+internal/\mbox{\hyperlink{__internal_2exceptions_8py}{exceptions.\+py}}\end{DoxyCompactItemize}
